src/codecoverage/CFCodeCoverageChecker.java

```diff
@@ -11,39 +11,73 @@
 	
 	static FileInfoService fileInfoService = ServiceFactory.getFileInfoService();
 	
-	public static void main(String[] args) {
+	public static void main(String[] args) throws Exception{
 		
 		System.out.println("This is the main section ");
 		
 		//STEP 1: be able to pass Slatwall directory as an argument from command line
 		
 		String slatwallDirectoryPath = args[0];
-		
-		//STEP 2: read all file names in /model folder store info in an array of file objects 
-		//		  [
-		//			{
-		//				filename:"Product.cfc",
-		//				path:"/entity"
-		//				functions:["getAvailableForPurchaseFlag","..."]
-		//			}
-		//		  ] 
-		//l
+	
 		String modelPath = slatwallDirectoryPath.concat("/model/");
 		modelPath.replaceAll("/", File.separator);
 		System.out.println(modelPath);
+		
+
+        File f = new File("htmlOutput.html");
+        BufferedWriter bw = new BufferedWriter(new FileWriter(f));
+       
+        bw.write("<html>");
+        bw.write("<body>");
+        bw.write("<h1>Report For Code Coverage In Slatwall</h1>");
+      //  bw.write("<textarea cols=120 rows=30>")
+      //  bw.close();
+        bw.write("<table>");
+        bw.write("<tr>");
```

Comment:
can you make this first row(<tr>) part of a <thead> and then all following rows part of <tbody>?